pkg/operator/starter.go

```diff
@@ -41,19 +41,22 @@ func RunOperator(clientConfig *rest.Config, stopCh <-chan struct{}) error {
 	ensureOperatorConfigExists(operatorConfigClient.OpenshiftcontrollermanagerV1alpha1(), "v3.11.0/openshift-controller-manager/operator-config.yaml")
 
 	operatorConfigInformers := operatorclientinformers.NewSharedInformerFactory(operatorConfigClient, 10*time.Minute)
-	kubeInformersNamespaced := informers.NewFilteredSharedInformerFactory(kubeClient, 10*time.Minute, targetNamespaceName, nil)
+	kubeInformersForOpenshiftOpenshiftControllerManagerNamespace := informers.NewSharedInformerFactoryWithOptions(kubeClient, 10*time.Minute, informers.WithNamespace(targetNamespaceName))
+	kubeInformersForOpenshiftCoreOperatorsNamespace := informers.NewSharedInformerFactoryWithOptions(kubeClient, 10*time.Minute, informers.WithNamespace(operatorNamespaceName))
```

Comment:
where is this informer started.

Reply:
Thanks, added a call to its Start method below